packages/core/src/operator/operator-runtime/bootOperatorRuntime.ts

```diff
@@ -52,6 +54,15 @@ export const bootOperatorRuntime = async (
 
         // If the referee is V2
         if (refereeIsV2) {
+            // If referee calculations address is not set, set it
+            if(operatorState.refereeCalculationsAddress === ""){
+                try {
+                    operatorState.refereeCalculationsAddress = await getRefereeCalculationsAddress();
```

Comment:
Please wrap this in a retry

Reply:
Added retry to getRefereeCalculationsAddress